### PR TITLE
Release v2.7.0

### PR DESCRIPTION
**Pull request by zhiqiangjin**
### Notes:
- Supported Controller version: **UserConnect-5.1.935**
- Supported Terraform version: **v0.12.x**

### Features:
1. Added support of attaching/detaching fireNet vpc to/from TGW in **aviatrix_aws_tgw_vpc_attachment**
2. Added support to create aviatrix_vpc of GCP provider in **aviatrix_vpc**
3. Added support of ``custom_saml_request_template`` in **aviatrix_saml_endpoint**
4. Added support of ``customized_routes`` and ``disable_local_route_propagation`` in **aviatrix_aws_tgw** 
5. Added option of retries for ``save`` or ``synchronize`` in **data_source_aviatrix_firenet_vendor_integration**

### Enhancements:
1. Reverted separating ``subnets`` to ``public_subnets`` and ``private_subnets`` in **aviatrix_vpc**
2. Changed calling ``update_access_policy`` from get to post in **aviatrix_firewall**

### Bug Fixes:
1. Fixed issue where it is unable to disable gateway ``split_tunnel``
2. Fixed issue where terraform refresh is not working for firewall policy
3. Fixed ``subnets`` reorder issue after import in **aviatrix_vpc**
4. Fixed the issue where creating with special characters causes parsing issue in **aviatrix_firewall_instance**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
-## 2.6.1 (Unreleased)
+## 2.7.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.1.935**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: Latest release is 5.1.973

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Added support of attaching/detaching fireNet vpc to/from TGW in **aviatrix_aws_tgw_vpc_attachment**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: - "Added support FOR ..."
- Change to "FireNet" and "VPC"

**Review by arvindsree — APPROVED** [review]
otherwise fine apart from the changes Anthony mentioned

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+2. Added support to create aviatrix_vpc of GCP provider in **aviatrix_vpc**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: "Added support for creating GCP VPC with ..."

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,26 @@
...
+3. Added support of ``custom_saml_request_template`` in **aviatrix_saml_endpoint**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: "Added support for ..."

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,26 @@
...
+4. Added support of ``customized_routes`` and ``disable_local_route_propagation`` in **aviatrix_aws_tgw**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: "Added support for ..."

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,26 @@
...
+### Features:
+1. Added support of attaching/detaching fireNet vpc to/from TGW in **aviatrix_aws_tgw_vpc_attachment**
+2. Added support to create aviatrix_vpc of GCP provider in **aviatrix_vpc**
+3. Added support of ``custom_saml_request_template`` in **aviatrix_saml_endpoint**
+4. Added support of ``customized_routes`` and ``disable_local_route_propagation`` in **aviatrix_aws_tgw** 
+5. Added option of retries for ``save`` or ``synchronize`` in **data_source_aviatrix_firenet_vendor_integration**
+
+### Enhancements:
+1. Reverted separating ``subnets`` to ``public_subnets`` and ``private_subnets`` in **aviatrix_vpc**
+2. Changed calling ``update_access_policy`` from get to post in **aviatrix_firewall**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: "GET to POST"

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,26 @@
...
+
+### Bug Fixes:
+1. Fixed issue where it is unable to disable gateway ``split_tunnel``
+2. Fixed issue where terraform refresh is not working for firewall policy
+3. Fixed ``subnets`` reorder issue after import in **aviatrix_vpc**
+4. Fixed the issue where creating with special characters causes parsing issue in **aviatrix_firewall_instance**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: - Fixed issue where **aviatrix_gateway** was unable to disable ``split_tunnel``
- ...
- Fixed issue where **aviatrix_vpc** ``subnets`` were reordering after an import
- ...

File: CHANGELOG.md (hunk 5 above)
Comment (inc1t3Ful) [inline CHANGELOG.md]: "Added support for ..."

**Pull request merged**